src/cities/cityConfigs.js

```diff
@@ -12,7 +12,7 @@ const configs = {
   }),
   bayreuthwohnraum: new CityConfig({
     cmsName: 'bayreuthwohnraum',
-    hostname: 'bayreuth.wohnen.integreat-app.de',
+    hostname: 'wohnraumboerse.bayreuth.de',
```

Comment:
I think we should wait until the DNS record is there. Maybe they want **www.**wohnraumboerse.bayreuth.de

Reply:
Hmm... true :+1:

Reply:
funnily, neuburg don't redirect www.raumfrei.neuburg-schrobenhausen.de to raumfrei.neuburg-schrobenhausen.de... Should we do sth about that (aka remind them that they could do sth about it)?

Reply:
@Schedulaar I think we do not need to do this. What do you think, @dkehne ?